assets/src/stories-editor/plugins/export.js

```diff
@@ -0,0 +1,57 @@
+/* global fetch, alert */
+
+/**
+ * WordPress dependencies
+ */
+import { PluginMoreMenuItem } from '@wordpress/edit-post';
+const { ampStoriesExport, FormData, URL } = window;
+
+const handleExport = () => {
+	const fromData = new FormData();
+
+	// Add the form data.
+	fromData.append( 'action', ampStoriesExport.action );
+	fromData.append( '_wpnonce', ampStoriesExport.nonce );
+	fromData.append( 'post_ID', document.getElementById( 'post_ID' ).value );
```

Comment:
A more safe approach for getting the post ID would be this:

```js
/**
 * WordPress dependencies
 */
import { select } from '@wordpress/data';

const { getCurrentPostId } = select;

fromData.append( 'post_ID', getCurrentPostId() );
```

Reply:
I wasn't aware of that package, thanks!